source/index.txt

```diff
@@ -32,145 +38,351 @@
-serverless cloud functions, declarative data access rules, and more. You can use
-{+service-short+} to write and host a full application in a managed cloud
-environment backed by MongoDB Atlas.
+.. introduction::
+   
+   MongoDB Atlas includes a suite of fully-managed services that power
+   user-facing applications and automated systems. You can use the
+   services to augment or replace a traditional application server.
 
-.. button:: Create an Account
-   :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+   Services let you write the code and configuration that matters for
+   your app without worrying about how to provision and manage the
+   infrastructure it runs on.
+   
+   .. button:: Start Building
+      :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+   
+   .. note:: Usage-Based Pricing & Free Tier
+      
+      Application services have usage-based billing and include a generous monthly
+      free tier for small development and production apps, so you can get started
+      at no cost. For more details, see :doc:`Billing </billing>`.
 
+.. image:: /images/plp/logic-tech.png
+   :alt: Atlas App Services Illustration
 
-.. card-group::
-   :style: extra-compact
-   :columns: 2
+.. kicker:: Access Your Data
 
-   .. card::
-      :headline: Create a New {+app+}
-      :url: https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-ui/
-      :icon: /images/icons/realm/cross-platform.svg
-      :icon-alt: {+app+} Icon
-      
-      {+apps+} contain all of your hosted services. Learn how to set up your
-      Atlas cluster and create a new serverless application.
+Read And Write Data From Any App With Powerful APIs
+---------------------------------------------------
 
-   .. card::
-      :headline: Build a Serverless App Backend
-      :url: https://www.mongodb.com/docs/atlas/app-services/tutorial/backend/
-      :icon: /images/icons/realm/realm-lightning.svg
-      :icon-alt: DB Triggers Icon
-      
-      See how to use HTTPS endpoints and triggers to run a serverless GitHub issue
-      tracker.
+Services have native access to data in your Atlas clusters and let you
+expose secure APIs to work with that data.
 
-   .. card::
-      :headline: Explore Template Applications
-      :url: https://www.mongodb.com/docs/atlas/app-services/template-apps/
-      :icon: /images/icons/realm/realm-mobile.svg
-      :icon-alt: {+service-short+} Mobile Icon
-      
-      Explore working applications to learn how to implement {+sync+} in 
-      mobile applications or use App Services.
+.. tabs::
 
-   .. card::
-      :headline: Learn More About Services
-      :url: https://www.mongodb.com/docs/atlas/app-services/introduction/
-      :icon: /images/icons/realm/graphql.svg
-      :icon-alt: GraphQL Icon
+   .. tab:: Device Sync
+      :tabid: device-sync
+
+      .. procedure::
+
+         .. step:: Create Your Data Model
```

Comment:
For all the services/steps, I did a pass on this doc on how to best talk about these services from someone who is coming in from Atlas's POV/making the steps a bit more contextual to the level of knowledge they may have about document database. Let me know what you think https://docs.google.com/document/d/1aEHckpA9AIg-VWN3apd23ne12AHbn5c_6U9G9uQ3flA/edit#